utils/text_norm_whisper_like/__init__.py

```diff
@@ -15,4 +15,4 @@ def get_txt_norm(txt_norm):
     elif txt_norm == "chime8":
```

Comment:
Could we add a doc (e.g. in __init__.py) explaining the set of differences and motivations w.r.t original Whisper?
I think people would like to know, and also we'll probably revisit this normalization sometime and it's good to have a picture of the considerations.
The number normalization is still off, right? That's one thing we'll probably have to think about. Currently Whisper is penalized for it every time a number appears in notsofar data.